spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageProtocolResolver.java

```diff
@@ -64,6 +64,6 @@ public Resource resolve(String location, ResourceLoader resourceLoader) {
 		if (location.startsWith("gs://")) {
 			return new GoogleStorageResource(this.storage, location);
 		}
-		return this.delegate.getResource(location);
+		return null;
```

Comment:
Thank you @artembilan for helping out with this. I'm as guilty as someone can be, but to be honest what would be best course of action now? I had only one commit for every changes, so reverting back is a lot of work. I could edit that file again, and add on a new commit, but history would really reflect the change. So what would be a good choice at this point?